packages/workers/src/main.ts

```diff
@@ -9,9 +9,16 @@ if (!process.env.CACHE_DATABASE_URL) {
 
 logger.info('Starting workers')
 
+const defiProvider = new DefiProvider()
```

Comment:
Declaring it inside the `runner` meant multiple initializations, each of them repeating certain logs.